docs/spicedb/operator.md

```diff
@@ -121,16 +121,17 @@ There may be exceptions to this rule, but they will be documented in release not
-| extraPodLabels               | A set of additional labels to add to the spicedb pods.                                                                                                                | string or map[string]string  |
+| Flag                         | Description                                                                                                                                                          | Type                        |
+| ---------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------- |
+| image                        | A specific container image to run.                                                                                                                                   | string                      |
+| replicas                     | The number of nodes to run for this cluster.                                                                                                                         | string or int               |
+| skipMigrations               | If true, the operator will not run migrations on changes to this cluster.                                                                                            | string or bool              |
+| tlsSecretName                | The name of a Kubernetes secret in the same namespace to use as the TLS credentials for SpiceDB services.                                                            | string                      |
+| dispatchUpstreamCASecretName | The name of a Kubernetes secret in the same namespace to use as the TLS CA validation. This should be the CA cert that was used to issue the cert in `tlsSecretName` | string                      |
+| datastoreTLSSecretName       | The name of a Kubernetes secret containing a TLS secret to use when connecting to the datastore.                                                                     | string                      |
+| spannerCredentials           | The name of a Kubernetes secret containing credentials for talking to Cloud Spanner. Typically, this would not be used, in favor of workload identity.               | string                      |
+| extraPodLabels               | A set of additional labels to add to the spicedb pods.                                                                                                               | string or map[string]string |
+| extraPodAnnotations          | A set of additional annotations to add to the spicedb pods.                                                                                                          | string or map[string]string |
```

Comment:
this reminds me we haven't made the new release including this feature 👀 
https://github.com/authzed/spicedb-operator/releases

Reply:
D'oh. Yeah, we need to do that.